src/applications/personalization/profile/components/edit/Edit.jsx

```diff
@@ -61,10 +62,17 @@ export const Edit = () => {
 
   const fieldInfo = getFieldInfo(query.get('fieldName'));
 
-  const returnRouteInfo = getRouteInfoFromPath(
-    query.get('returnPath'),
-    routesForNav,
-  );
+  const returnRouteInfo = (() => {
+    try {
+      return getRouteInfoFromPath(query.get('returnPath'), routesForNav);
+    } catch (e) {
+      // default to using the root route if the returnPath is invalid
```

Comment:
probably not, mostly hoping this would be something that would blow up tests, or blow up the local runtime with errors (in my case it actually blew up my tests, so I updated the code accordingly)